epm/Distribution/Client/Install.hs

```diff
@@ -1052,6 +1052,7 @@ performInstallations verbosity
     }
     reportingLevel = fromFlag (installBuildReports installFlags)
     logsDir        = fromFlag (globalLogsDir globalFlags)
+    etaPatchesDir = fromFlagOrDefault [] (installEtaPatchesDirectory installFlags)
```

Comment:
Why not set the default to `defaultPatchesDir` right here? You'll have to put it inside the monadic context as opposed to the `where` clause.